lms/static/sass/elements/_controls.scss

```diff
@@ -404,7 +404,7 @@
     }
   }
 
-  .btn-link {
+  .btn-secondary {
```

Comment:
This is something we added in an earlier teams story that I haven't had a chance to review / cleanup, but now is a good opportunity I believe. We currently use this in the teams details sidebar for "Leave Team" if I'm not mistaken. 

Instead of using the btn-secondary name here, which is used already for other pattern library button styles above, we should update this btn-link styling to echo the UX button style that has no padding to the left / right.

This is a link to the "btn-link" pattern library: https://github.com/edx/ux-pattern-library/blob/master/_src/pattern-library/sass/components/_buttons.scss#L229

![image](https://cloud.githubusercontent.com/assets/2023680/9411499/3a5bd936-47f4-11e5-849b-8644bf1896a8.png)